CHANGES.rst

```diff
@@ -5,8 +5,11 @@
 6.1 (unreleased)
 ================
 
+- Drop support for Python 3.8.
+
```

Comment:
No preliminary support / testing for Python 3.14?

Reply:
That was already in there: https://github.com/zopefoundation/ExtensionClass/commit/781ea473993631392e68f7373b75c37d041a45e6